### PR TITLE
Refactor: move Footer and MatomoWrapper to components-package

### PR DESCRIPTION
**Pull request by nenonja**
## Description
Pre-work before implementing error pages. Move `Footer` component and `MatomoWrapper` to `components` package
### Closes

**[DEV-XXX](https://helsinkisolutionoffice.atlassian.net/browse/DEV-XXX):**

### CODE REVIEW

**Comment by terovirtanen** [comment]
**$(upper(replace(parameters.proxy,'-', ' '))) is deployed to platta: https://events-graphql-proxy-pr207.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**$(upper(replace(parameters.proxy,'-', ' '))) is deployed to platta: https://venue-graphql-proxy-pr207.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**$(upper(replace(parameters.proxy,'-', ' '))) is deployed to platta: https://events-graphql-proxy-pr207.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**$(upper(replace(parameters.proxy,'-', ' '))) is deployed to platta: https://venue-graphql-proxy-pr207.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://harrastukset-pr207.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**Hobbies-Helsinki branch is deployed to platta: https://harrastukset-pr207.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**$(upper(replace(parameters.proxy,'-', ' '))) is deployed to platta: https://venue-graphql-proxy-pr207.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://harrastukset-pr207.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**$(upper(replace(parameters.proxy,'-', ' '))) is deployed to platta: https://events-graphql-proxy-pr207.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Events-Helsinki branch is deployed to platta: https://tapahtumat-pr207.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Hobbies-Helsinki branch is deployed to platta: https://harrastukset-pr207.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Sports-Helsinki branch is deployed to platta: https://liikunta-pr207.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://tapahtumat-pr207.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**$(upper(replace(parameters.proxy,'-', ' '))) is deployed to platta: https://events-graphql-proxy-pr207.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://harrastukset-pr207.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://liikunta-pr207.dev.hel.ninja :laughing::tada::tada::tada:

**Review by melniiv — APPROVED** [review]
LGTM

File: packages/components/src/constants/constants.ts [hunk 1]
```diff
@@ -59,3 +60,27 @@ export const FEEDBACK_LINKS = {
...
+export const DEFAULT_HEADER_MENU_NAME: Record<AppLanguage, string> = {
+  fi:
+    process.env.NEXT_PUBLIC_CMS_HEADER_MENU_NAME_FI ??
+    'Events Helsinki Header FI',
+  en:
+    process.env.NEXT_PUBLIC_CMS_HEADER_MENU_NAME_EN ??
+    'Events Helsinki Header EN',
+  sv:
+    process.env.NEXT_PUBLIC_CMS_HEADER_MENU_NAME_SV ??
+    'Events Helsinki Header SV',
```
Comment (nikomakela) [inline packages/components/src/constants/constants.ts:73]: Events Helsinki Header is the prefix of the name of the header in Events app. If this is now generic, also the default name should be something more generic, like "Header FI", "Header SV", etc. There is currently also an option in the CMS to search a menu with "ID"-parameter, but the ID-value is something that Geniem defines per instance, so we cannot be sure whether it's the same between the multiple app CMS-instances. The name is something that we can self design and set, so it's better to use that one.

**Pull request merged**